corgi/core/models.py

```diff
@@ -916,6 +916,20 @@ class Namespace(models.TextChoices):
         UPSTREAM = "UPSTREAM"
         REDHAT = "REDHAT"
 
+    class Arch(models.TextChoices):
+        aarch64 = "aarch64"
+        i386 = "i386"
+        ia64 = "ia64"
+        i686 = "i686"
```

Comment:
This is almost alphabetical order (thanks for that), except for these two lines:
```suggestion
        i686 = "i686"
        ia64 = "ia64"
```